app/scripts/metamask-controller.js

```diff
@@ -1699,6 +1706,7 @@ export default class MetamaskController extends EventEmitter {
       SwapsController: this.swapsController.store,
       EnsController: this.ensController.store,
       ApprovalController: this.approvalController,
+      LoggingController: this.loggingController,
```

Comment:
We don't want the `LoggingController` here as we want to persist all the logs.

We also need to ensure we reference it in both the persisted store and mem store to ensure logs are saved and visible to the frontend.